app/assets/stylesheets/common/_markus.scss

```diff
@@ -179,15 +180,15 @@ textarea {
   }
 
   &[disabled] {
-    background-color: $disabled-grey;
-    border-color: $disabled-dark;
-    color: $disabled-dark;
+    background-color: $background-support;
+    border-color: $background-support;

```

Comment:
I think this one does need a different colour, otherwise the border will disappear